.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: python
 jobs:
   include:
+    - python: "3.9"
+      env: TOXENV=py39
     - python: "3.8"
-      env: TOXENV=black,flake8
+      env: TOXENV=black,flake8  # as soon as version 20.8b1 or higher is released, we can move it to py39
```

Comment:
20.8b1 is already released. We need a higher version than that (https://pypi.org/project/black/20.8b1/)

Reply:
changed